_posts/en/newsletters/2024-10-18-newsletter.md

```diff
@@ -0,0 +1,117 @@
+---
+title: 'Bitcoin Optech Newsletter #325'
+permalink: /en/newsletters/2024/10/18/
+name: 2024-10-18-newsletter
+slug: 2024-10-18-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME:harding
+
+## News
+
+_No significant news this week was found in any of our [sources][optech sources]._
+
+## Changes to services and client software
+
+*In this monthly feature, we highlight interesting updates to Bitcoin
+wallets and services.*
+
+- **Coinbase adds taproot send support:**
+  Coinbase exchange [now supports][coinbase post] user withdrawals (send) to taproot
+  [bech32m][topic bech32] addresses.
+
+- **Dana wallet released:**
+  [Dana wallet][dana wallet github] is a [silent payment][topic silent payments] wallet focused the
+  donation use case. The developers recommend using [signet][topic signet] and
+  also run a signet [faucet][dana wallet faucet].
+
+- **Bitcoin script editor and visualizer:**
+  The Saving Satoshi [script editor][saving satoshi editor] allows for visually
+  building and executing Bitcoin scripts.
```

Comment:
I am not sure we should include this just yet, I've played with the editor for a few minutes and already found serious bugs in the script interpreter. For example, `OP_0 OP_SHA256` does not hash the empty byte array but the string "0". Additionally, only around half the existing opcodes are implemented, for example most arithmetic and stack opcodes are missing. And there are other issues, for example cleanstack is enforced without explaining why the script "failed".

Reply:
This is good feedback. Will remove for this month.